gradle/libs.versions.toml

```diff
@@ -1,6 +1,6 @@
 [versions]
 annotationApiVersion = "2.1.1"
-antlrVersion = "2.7.7"
+antlrVersion = "2.7.6"
```

Comment:
why are we demoting this version?

Reply:
Sorry, version catalog thinks 2.7.7 doesn't exist for some reason. I fixed it.